### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Aegide**
I didn't know this mod existed until i saw Botania use "emi" in the tags, it looks amazing.

### CODE REVIEW

File: src/main/resources/assets/emi/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,540 @@
+{
+  "key.keyboard.control": "Ctrl",
+  "key.keyboard.alt": "Alt",
+  "key.keyboard.shift": "Shift",
```
Comment (LambdAurora) [inline src/main/resources/assets/emi/lang/fr_fr.json:4]:
```suggestion
  "key.keyboard.shift": "Maj",
```

File: src/main/resources/assets/emi/lang/fr_fr.json [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+  "tag.c.nether_stars": "Étoiles du Nether",
+  "tag.c.netherrack": "Netherrack",
+  "tag.c.obsidian": "Obsidienne",
+  "tag.c.pickaxes": "Pioches",
+  "tag.c.potions": "Potions",
+
+  "tag.c.red_sandstone": "Grès rouge",
+  "tag.c.red_sandstone_slabs": "Dalles en grès rouge",
+  "tag.c.red_sandstone_stairs": "Escaliers en grès rouge",
+  "tag.c.rods.blaze": "Bâton de Blaze",
```
Comment (LambdAurora) [inline src/main/resources/assets/emi/lang/fr_fr.json:509]:
```suggestion
  "tag.c.rods.blaze": "Bâtons de Blaze",
```

File: src/main/resources/assets/emi/lang/fr_fr.json [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+  "emi.view_all_recipes": "Voir toutes les recettes",
+  "emi.tree_welcome": "Bienvenue dans %s",
+  "emi.no_tree": "Vous n'avez pas encore défini de graph",
+  "emi.random_tree": "Vous voulez-vous en voir un au hasard ?",
+  "emi.random_tree_input": "Appuyez sur §6[ctrl + r]",
+
+  "tooltip.emi.fill_recipe": "Utiliser les ingrédients",
+  "tooltip.emi.fill_recipe_no_server": "EMI n'est pas installé sur ce serveur",
+  "tooltip.emi.set_default": "Définir par défaut",
+  "tooltip.emi.view_tree": "Voir graph",
```
Comment (LambdAurora) [inline src/main/resources/assets/emi/lang/fr_fr.json:161]:
```suggestion
  "tooltip.emi.view_tree": "Voir graphe",
```

**Review by LambdAurora — COMMENTED** [review]
Not in the mood for a full review, but here's some of the stuff I noticed.
Also the spacing on parenthesis feel super weird.

**Comment by Aegide** [comment]
@LambdAurora Oh, i didnt expect someone to actually care about my PR and to proofread it, thanks for your time.

**Comment by Aegide** [comment]
Ok, the translation should be complete.

**Pull request merged**